app/views/admin/school_onboardings/email/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Change email address for <%= @school_onboarding.school_name %></h1>
+
+<%= simple_form_for([:admin, @school_onboarding], url: admin_school_onboarding_email_path(@school_onboarding)) do |f| %>
+
+  <%= render 'shared/errors', subject: @school_onboarding, subject_name: 'form' %>
```

Comment:
Do we need this with simple form? I don't think we use it elsewere.

Reply:
No we don't I turned this from a form_for to a simple_form_for and should have taken out that bit...